mmv1/products/firebasehosting/terraform.yaml

```diff
@@ -63,6 +63,89 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           channel_id: channel-full
         test_env_vars:
           project_id: :PROJECT_NAME
+  Release: !ruby/object:Overrides::Terraform::ResourceOverride
+    import_format: [
+      'sites/{{site_id}}/releases/{{release_id}}',
+      'sites/{{site_id}}/channels/{{channel_id}}/releases/{{release_id}}',
+      '{{site_id}}/{{release_id}}',
+      '{{site_id}}/{{channel_id}}/{{release_id}}'
+    ]
+    skip_delete: true # can't delete a release
```

Comment:
I want to confirm that we won't have orphaned resources from running these below examples as tests. They should be eventually deleted once the parent "site" is removed right?

Reply:
Yes, and even for the default hosting site, which can't be deleted, there is a retainedReleaseCount field (default 10) that specifies automatically deletes the oldest release if full. https://firebase.google.com/docs/reference/hosting/rest/v1beta1/sites.channels#Channel.FIELDS.retained_release_count